src/cmd/src/cli/bench/metadata.rs

```diff
@@ -17,7 +17,9 @@ use std::time::Instant;
 use common_meta::key::TableMetadataManagerRef;
 use common_meta::table_name::TableName;
 
-use super::{bench_self_recorded, create_region_routes, create_table_info};
+use super::{
+    bench_self_recorded, create_region_routes, create_region_wal_options, create_table_info,
+};
```

Comment:
nit(non-block):  There are some tech debts.

Let's follow the style guide.

> Avoid glob imports. Only use a glob import when you mean “import lots of items from some module”, not when you mean “I want lots of items from this module, but I don’t want to write them out”.
It is common to import all of a prelude module using a glob import (e.g., use futures::prelude::*').
Exception, most test modules should have a use super::*; import.

https://pingcap.github.io/style-guide/rust/modules.html

Reply:
Changed to import glob.